src/comparator/wrappers.py

```diff
@@ -138,6 +138,7 @@ class Field:
     message_resource: message-level resource definition.
     api_version: the version of the API definition files.
     map_entry: type of the field if it is a map.
+    oneof_name: set if the field is in oneof.
```

Comment:
This comment isn't quite accurate anymore. The `oneof_name` is now an optional string that contains the name of the oneof the field belongs to, if it belongs to one.

Reply:
Yes, the oneof_name is optional, which is default to be `None` in the below init function. If it is in oneof, from the outer message, we will assign the corresponding oneof name to it. 

Modified the comment to `oneof_name: the oneof name that the field belongs to if it is in any oneof.` Please let me know if that is not appropriate. Thanks

Reply:
Yes, that's better. Thanks!